### ISSUE TITLE
Laptop's sound goes quiet when loading Makie (on Windows)

### ISSUE THREAD

**Issue by cserteGT3** [JuliaPlots/Makie.jl#314]
Every time I load the package, the sound of my laptop goes away for a few seconds, then it comes back.
My laptop is a ThinkPad Edge E431 and the headphone I use is connected to it via a ThinkPad OneLink Dock. I don't really know how to trace this bug, but happy to try anything.
Attached the versioninfos, what other informations would be useful?
OS: Windows 10 Enterprise, build: 17763.316
```julia
julia> versioninfo()
Julia Version 1.1.0
Commit 80516ca202 (2019-01-21 21:24 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: Intel(R) Core(TM) i7-3632QM CPU @ 2.20GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, ivybridge)
Environment:
  JULIA_NUM_THREADS = 4

(v1.1) pkg> st
    Status `C:\Users\cstamas\.julia\environments\v1.1\Project.toml`
  [537997a7] AbstractPlotting v0.9.5 #master (https://github.com/JuliaPlots/AbstractPlotting.jl.git)
  [c52e3926] Atom v0.7.14
  [6e4b80f9] BenchmarkTools v0.4.2
  [5ae59095] Colors v0.9.5
  [5789e2e9] FileIO v1.0.5
  [f7f18e0c] GLFW v2.3.0
  [e9467ef8] GLMakie v0.0.5 #master (https://github.com/JuliaPlots/GLMakie.jl.git)
  [4d00f742] GeometryTypes v0.7.3
  [7073ff75] IJulia v1.17.0
  [6218d12a] ImageMagick v0.7.1
  [c601a237] Interact v0.9.1
  [033835bb] JLD2 v0.1.2
  [e5e0dc1b] Juno v0.5.4
  [fc18253b] LazyJSON v0.1.0
  [ee78f7c6] Makie v0.9.2 #master (https://github.com/JuliaPlots/Makie.jl.git)
  [dbd62bd0] MakieGallery v0.0.4
  [283c5d60] MeshCat v0.6.0
  [6ad125db] MeshCatMechanisms v0.3.0
  [66fc600b] ModernGL v1.0.0
  [510215fc] Observables v0.2.3
  [9b87118b] PackageCompiler v0.6.3+ #master (https://github.com/JuliaLang/PackageCompiler.jl.git)
  [df47a6cb] RData v0.6.0
  [ce6b1742] RDatasets v0.6.1
  [dee08c22] RegionTrees v0.2.0
  [295af30f] Revise v1.1.0
  [366cf18f] RigidBodyDynamics v1.4.0
  [6038ab10] Rotations v0.11.1
  [90137ffa] StaticArrays v0.10.3
  [65254759] StatsMakie v0.0.2
  [8603256b] ZChop v0.3.3
```

**Comment by asinghvi17** [comment]
Does this also happen when loading other packages (you could try `Plots`)?

**Comment by cserteGT3** [comment]
Did not experience it by other packages. I tried it now with `Plots`, also works fine.
Sometimes the sound also goes quiet when I first open a scene with Makie (the first call in a session that creates a `Scene`).
Could it be due to some OpenGL glitch?

**Comment by asinghvi17** [comment]
Possibly, do you have two graphics cards?  (Or one integrated and one dedicated)

**Comment by cserteGT3** [comment]
Yes, an integrated: Intel® HD Graphics 4000, and a dedicated: Nvidia GeForce GT 740M.
Tried with a completely different laptop (a Sony Vaio SVF1521H1E), with an Intel Core i3-3227U (integrated Intel HD 4000) and a GeForce GT 740M (also on Windows 10) and everything worked fine.

**Comment by asinghvi17** [comment]
Have you updated your drivers recently?  (Specifically for the Intel one, as I'm not sure if it supports OpenGL.

**Comment by cserteGT3** [comment]
The issue started some weeks ago and unfortunately I don't remember if it started after a driver or windows update.

**Comment by SimonDanisch** [comment]
try GLMakie#master & AbstractPlotting#master! I mostly fixed #315, which might be related to your issue (not refreshing the window and freezing OpenGL might do some bad stuff for badly written drivers - which is my only explanation of what might be going on here).

**Comment by cserteGT3** [comment]
Sorry for the late response, I tried to recreate my issue. Meanwhile I had a Windows update and also a Lenovo one (possibly containing driver updates). Now the sound issue is gone on the tagged and #master versions too.
Thank you for your help!
